### PR TITLE
Step towards automation

### PR DESCRIPTION
**Pull request by dakaa16**
The following changes are a suggestion on how to automate pull requests, and thus minimizing management time, and approval time for pull requests.

A GitHub Action has been set up to run tests and approvalTests when pushing a branch. Note; currently any pushed branch will start this Action. Tests will run on Ubuntu 18.04, Windows Server 2019 and macOS 10.15, all on both JDK 8 and 11.

Some tests have been corrected, in order for all tests to succeed on all above mentioned platforms and Java versions.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -0,0 +1,8 @@
+artifactory_url = https://artifactory.tools.bdpnet.dk/artifactory
```
Comment (Rune-Christensen) [inline gradle.properties:1]: This line is wrong, points to internal artifactory.

File: gradle.properties [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+org.gradle.daemon=false 
+
+systemProp.sonar.login=1434c55ed28e06d8240c2347c94ee0306484eb93
+systemProp.sonar.host.url=http://sonar.devops.bdpnet.dk
```
Comment (Rune-Christensen) [inline gradle.properties:5]: This line is wrong, points to internal sonarqube

**Review by Rune-Christensen — CHANGES_REQUESTED** [review]
gradle.properties should not point to internal products. 
This should be fixed before merge.

**Comment by dakaa16** [comment]
@Rune-Christensen somehow the code you are referencing got included in an attempt to sign off a commit. I'll be deleting this pull request and create a new one.

**Pull request closed without merging**